src/routers/transaction.ts

```diff
@@ -446,4 +446,42 @@ export function handle(context: IndexerContext, router: Router) {
             }
         }
     );
+
+    /**
+     * @swagger
+     * /aggs-tx/count:
+     *   get:
+     *     summary: Returns count of each transaction type in the blockNumber
+     *     tags: [Transaction]
+     *     parameters:
+     *       - name: blockNumber
+     *         description: filter by blockNumber
+     *         in: query
+     *         required: true
+     *         type: number
+     *     responses:
+     *       200:
+     *         description: count of each transaction type in the blockNumber
+     *         schema:
+     *           type: array
+     *           items:
+     *             $ref: '#/definitions/Transaction'
+     */
+    router.get(
+        "/aggs-tx/count",
```

Comment:
I think this name isn't great. I can't expect what will be aggregated

Reply:
IMO, this method can be located in the Block router because the `blockNumber` is required.